pkgs/desktops/gnome/extensions/default.nix

```diff
@@ -59,8 +59,9 @@ in rec {
   gnome40Extensions = mapUuidNames (produceExtensionsList "40");
   gnome41Extensions = mapUuidNames (produceExtensionsList "41");
   gnome42Extensions = mapUuidNames (produceExtensionsList "42");
+  gnome43Extensions = mapUuidNames (produceExtensionsList "43");
 
-  gnomeExtensions = lib.trivial.pipe (gnome40Extensions // gnome41Extensions // gnome42Extensions) [
+  gnomeExtensions = lib.trivial.pipe (gnome40Extensions // gnome41Extensions // gnome42Extensions // gnome43Extensions) [
```

Comment:
Do we want to throw some of the older stuff out?

Reply:
Personally, I would just drop everything but the latest, only keeping two versions when transitioning.

Reply:
I personally don't see an issue with having all in there, since by default they will just not get loaded if you don't set `disable-extension-version-validation = true`. In some cases, old extensions keep working even if they are not built for a newer version. But 3 is probably enough?

Reply:
Three sounds reasonable, especially since this roughly coincides with GNOME's support frame.